aws/v16.2.0/release.diff

```diff
@@ -0,0 +1,73 @@
+# Generated with:                                                  # Generated with:
+# devctl release create --name v16.1.0 --provider aws --base 16 |  # /home/whites/workspace/devctl/devctl release create --name 16
+apiVersion: release.giantswarm.io/v1alpha1                         apiVersion: release.giantswarm.io/v1alpha1
+kind: Release                                                      kind: Release
+metadata:                                                          metadata:
+  annotations:                                                       annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp        giantswarm.io/docs: https://docs.giantswarm.io/reference/cp
+  creationTimestamp: null                                            creationTimestamp: null
+  name: v16.1.0                                                 |    name: v16.2.0
+spec:                                                              spec:
+  apps:                                                              apps:
+  - componentVersion: 1.4.0                                          - componentVersion: 1.4.0
```

Comment:
sorry again. nit, you also need to update the component version here 🙂

Reply:
done